### PR TITLE
MINOR: fix log package `AbstractIndex.scala` file comment typo

### PR DESCRIPTION
**Pull request by panguncle**
MINOR: fix log package `AbstractIndex.scala` file comment typo

### CODE REVIEW

**Review by chia7712 — APPROVED** [review]
@panguncle Thanks for your patch. LGTM

File: core/src/main/scala/kafka/log/AbstractIndex.scala [hunk 1]
```diff
@@ -89,7 +89,7 @@ abstract class AbstractIndex(@volatile private var _file: File, val baseOffset:
       lookup. So that, the entire warm section is really "warm".
       When doing warm-section lookup, following 3 entries are always touched: indexEntry(end), indexEntry(end-N),
       and indexEntry((end*2 -N)/2). If page size >= 4096, all the warm-section pages (3 or fewer) are touched, when we
-      touch those 3 entries. As of 2018, 4096 is the smallest page size for all the processors (x86-32, x86-64, MIPS,
+      touch those 3 entries. As of 2048, 4096 is the smallest page size for all the processors (x86-32, x86-64, MIPS,
```
Comment (ijuma) [inline core/src/main/scala/kafka/log/AbstractIndex.scala:92]: This change doesn't make sense. `2018` was meant to be the year.

Comment (chia7712) [reply core/src/main/scala/kafka/log/AbstractIndex.scala:92]: will revert it. sorry for rough merge :(

Comment (chia7712) [reply core/src/main/scala/kafka/log/AbstractIndex.scala:92]: #9751

**Pull request merged**